plugins/inputs/marklogic/README.md

```diff
@@ -0,0 +1,50 @@
+# Marklogic Plugin
+
+The Marklogic Telegraf plugin gathers status metrics from one or more host in the Marklogic cluster.
+
+### Configuration:
+
+```toml
+# Description
+[[inputs.marklogic]]
+
+## List URLs of Marklogic hosts using Management API endpoint.
+# hosts = ["http://localhost:8002/manage/v2/hosts/${hostname}?view=status&format=json"]
+
+# Using HTTP Digest Authentication.
+# digest_username = "telegraf"
+# digest_password = "p@ssw0rd"
```

Comment:
We should add support for the TLS configuration along with the usual options.

Reply:
:+1: